configs/cityscapes/README.md

```diff
@@ -0,0 +1,33 @@
+# Benchmark and Model Zoo
+
+## Environment
+
+### Hardware
+
+- 1 NVIDIA GeForce GTX 1080 Ti GPU
+
+## Common settings
+
+- All baselines were trained using 1 GPU with a batch size of 2 (2 images per GPU) using the [linear scaling rule](https://arxiv.org/abs/1706.02677) to scale the learning rate. The learning rate in the configs is set for a batch size of 16 to match the default of the coco models.
+- All models were trained on `cityscapes_train`, and tested on `cityscapes_val`.
+- 1x training schedule indicates 64 epochs which corresponds to slightly less than the 24k iterations reported in the original schedule from the [Mask R-CNN paper](https://arxiv.org/abs/1703.06870)
+- All pytorch-style pretrained backbones on ImageNet are from PyTorch model zoo.
+
+
+## Baselines
+
+Download links and more models with different backbones and training schemes will be added to the model zoo.
+
+
+### Faster R-CNN
+
+|    Backbone     |  Style  | Lr schd | Scale    | Pretraining | Mem (GB) | Train time (s/iter) | Inf time (fps) | box AP | Download |
+| :-------------: | :-----: | :-----: | :---:    | :---------: | :----:   | :----:              | :----:          | :----: | :------: |
+|    R-50-FPN     | pytorch |   1x    | 800-1024 | Backbone    | 4.9      | 0.345               | 8.8            | 36.0   | [model](https://open-mmlab.s3.ap-northeast-2.amazonaws.com/mmdetection/models/cityscapes/faster_rcnn_r50_fpn_1x_city_20190727-7b9c0534.pth) |
```

Comment:
Better to use Y/N for pretraining.

Reply:
I was trying to specify wether we used coco pretraining. Probably we should just remove this field until we have coco pretrained models and mention somewhere else, that we trained from scratch.

Reply:
Yes we can just remove this column. In this repo, pretraining usually means using the pretrained model from ImageNet and scratch means no pretrained weights for backbones. If we further add models pretrained from COCO and finetuned on CityScapes, we need to clearly indicate that.

Reply:
Yes. Let's think about that once we have the finetuned models.